src/github.js

```diff
@@ -220,11 +221,12 @@ function toString(eventsByRepository) {
 
 module.exports = (params = {}) => {
   const schema = Joi.object().keys({
-    username: Joi.string().alphanum().min(1).max(30).required(),
-    password: Joi.string().regex(/^[a-zA-Z0-9]{3,30}$/),
+    username: Joi.string().replace('-','_').token().min(1).max(30).required(), // .token() requires the string be alphanumeric or an underscore. github currently allows usernames with alphanumberics and hyphens
+    password: Joi.string().regex(/^[a-zA-Z0-9]{3,30}$/), // ? why do we need password?
```

Comment:
Bad code! 🚫 🔥

Reply:
before, using `alphanum()` on line 224 would not have worked for usernames with hyphens in them. are you saying that there is most likely a Joi method to do this more cleanly?

Reply:
I was trying to say that we don't need password